composer.json

```diff
@@ -1,38 +1,39 @@
 {
-    "name": "prestashop/autoupgrade",
-    "description": "PrestaShop module autoupgrade",
-    "homepage": "https://github.com/PrestaShop/autoupgrade",
-    "license": "AFL-3.0",
-    "authors": [
-        {
-            "name": "PrestaShop SA",
-            "email": "contact@prestashop.com"
-        }
-    ],
-    "require": {
-        "php": ">=5.6",
-        "ext-zip": "*",
-        "symfony/filesystem": "~2.8",
-        "doctrine/collections": "~1.3.0",
-        "twig/twig": "^1.35|^3"
+  "name": "prestashop/autoupgrade",
+  "description": "PrestaShop module autoupgrade",
+  "homepage": "https://github.com/PrestaShop/autoupgrade",
+  "license": "AFL-3.0",
+  "authors": [
+    {
+      "name": "PrestaShop SA",
+      "email": "contact@prestashop.com"
+    }
+  ],
+  "require": {
+    "php": ">=5.6",
```

Comment:
The PHP version here doesn't match with the module's dependency requiring PHP 7.2 minimum